### PR TITLE
crud

### PR DESCRIPTION
**Pull request by Kevalkukadiya**

### CODE REVIEW

File: rest_api/api/views.py [hunk 1]
```diff
@@ -216,3 +216,84 @@ class StudentRUD(RetrieveUpdateDestroyAPIView):
...
+        id = pk
+        if id is not None:
+            stu = Student.objects.get(id=id)
+            serializer = StudentSerializer(stu)
+            return Response(serializer.data)
+
+
+    def create(self, request):
+        serializer = StudentSerializer(data=request.data)
+        print(request.data)
```
Comment (Hardik0812) [inline rest_api/api/views.py:242]: remove print request

File: rest_api/rest_api/urls.py [hunk 2]
```diff
@@ -1,7 +1,21 @@
 from django.contrib import admin
-from django.urls import path
+from django.urls import path, include
 from api import views
 
+from rest_framework.routers import DefaultRouter
+
+router = DefaultRouter()
```
Comment (Hardik0812) [inline rest_api/rest_api/urls.py:7]: user path instead of router

File: rest_api/api/views.py (hunk 1 above)
Comment (Kevalkukadiya) [reply rest_api/api/views.py:242]: Done

File: rest_api/rest_api/urls.py (hunk 2 above)
Comment (Kevalkukadiya) [reply rest_api/rest_api/urls.py:7]: path = DefaultRouter() use this got error 
router.register('mstudentapi', views.StudentModelViewSet,basename='student')
 path('', include(router.urls)),
Error - router is not defined

Comment (Hardik0812) [reply rest_api/rest_api/urls.py:7]: Path means url pattern search in Google url pattern and set path in urls.py

Comment (Kevalkukadiya) [reply rest_api/rest_api/urls.py:7]: Sanje call ma samjve ne

**Pull request merged**